src/io.cr

```diff
@@ -259,7 +259,7 @@ abstract class IO
   end
 
   # Writes a formatted string to this IO.
-  # For details on the format string, see `Kernel::sprintf`.
+  # For details on the format string, see top-level `sprintf`.
```

Comment:
```suggestion
  # For details on the format string, see top-level `printf`.
```

Reply:
thanks! fixing...

Reply:
what's the policy/preference on squashing commits to clean up PRs?

Reply:
That doesn't work, that will point to `printf` in IO. You need to use `::printf`

Reply:
The policy is you don't care about how many commits are in here, we always squash PRs before merging.

Reply:
thanks! fixed in https://github.com/crystal-lang/crystal/pull/9549/commits/3ceb8ff62e8b163772fa2c1edf2293612a43b02f

Reply:
it doesn't generate a link to the top-level doc page, but it does prevent a link internal to the class

Reply:
Oh, maybe it's not implemented in the doc generator... I guess leaving it like that is good for now.